Userland/Libraries/LibJS/Runtime/Temporal/AbstractOperations.cpp

```diff
@@ -1666,41 +1666,30 @@ ThrowCompletionOr<TemporalTime> parse_temporal_time_string(VM& vm, String const&
     return TemporalTime { .hour = result.hour, .minute = result.minute, .second = result.second, .millisecond = result.millisecond, .microsecond = result.microsecond, .nanosecond = result.nanosecond, .calendar = move(result.calendar) };
 }
 
-// 13.38 ParseTemporalTimeZoneString ( isoString ), https://tc39.es/proposal-temporal/#sec-temporal-parsetemporaltimezonestring
-ThrowCompletionOr<TemporalTimeZone> parse_temporal_time_zone_string(VM& vm, String const& iso_string)
+// 13.38 ParseTemporalTimeZoneString ( timeZoneString ), https://tc39.es/proposal-temporal/#sec-temporal-parsetemporaltimezonestring
+ThrowCompletionOr<TemporalTimeZone> parse_temporal_time_zone_string(VM& vm, String const& time_zone_string)
 {
-    // 1. Let parseResult be ParseText(StringToCodePoints(isoString), TemporalTimeZoneString).
-    auto parse_result = parse_iso8601(Production::TemporalTimeZoneString, iso_string);
```

Comment:
I think `TemporalTimeZoneString` is unused after this, let's remove it from the public parsing interface

Reply:
Actually, it seems to be gone completely from the spec, I wonder where they removed it...

Reply:
Sure, would you prefer a separate commit?

Reply:
Aha, it's removed in https://github.com/tc39/proposal-temporal/commit/eec8efab, so just add that to the second commit :)

Reply:
done!